modules/smart-agent_redis/conf/07-memory-fragmentation-high.yaml

```diff
@@ -0,0 +1,28 @@
+module: redis
+name: high memory fragmentation ratio
+runbook_url: "https://www.dynatrace.com/news/blog/introducing-redis-server-monitoring/#key-metrics"
+
```

Comment:
```suggestion
tip: restart redis to recover memory previously unusable due to fragmentation or enable the new active defragmentation feature available since redis 4

```